src/main/java/com/example/bank/client/ClientRequest.java

```diff
@@ -0,0 +1,9 @@
+package com.example.bank.client;
+
+import com.example.bank.client.jpa.Address;
```

Comment:
czemu tu tyle pustych linii miedzy importami ? sformatuj to. A najlepiej caly codebase